easybuild/framework/easyblock.py

```diff
@@ -2667,6 +2678,18 @@ def build_and_install_one(ecdict, init_env, hooks=None):
         _log.debug("Stop set to %s" % stop)
         app.cfg['stop'] = stop
 
+    continue_from = build_option('continue')
+    if continue_from is not None:
+        _log.experimental("Continuing from step %s" % continue_from)
+        skipsteps = []
+        for step in build_option('valid_stops'):
+            if step != continue_from:
+                _log.debug("Adding %s to skipsteps in continue mode" % step)
+                skipsteps.append(step)
+            else:
+                break
+        app.cfg['skipsteps'] = skipsteps
```

Comment:
This may result in the `skipsteps` being added to the saved easyconfig file, which doesn't seem like a good idea to me...

Maybe introduce a `self.steps_to_skip`, which can be used both for `skipsteps` and `--continue`?